### PR TITLE
fix(company wallet): update active and inactive card style

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

update main active card style according to new design
differentiate wallet card based on status - active, inactive and unknown

## Why

not able to differentiate wallet cards

## Issue

#443 

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/pages/CompanyWallet/WalletCard.tsx [hunk 1]
```diff
@@ -30,50 +33,78 @@ export default function WalletCard({
...
-                <Typography variant="body2">
-                  {t('content.companyWallet.inactive')}
-                </Typography>
-              </div>
-            )}
+    <div className="wrapper-container">
+      <div
+        style={{
+          backgroundColor:
+            wallet?.credentialSubject[0].status === 'Active'
```
Comment (nidhigarg-bmw) [inline src/components/pages/CompanyWallet/WalletCard.tsx:50]: get this 'Active' from above interface type

File: src/components/pages/CompanyWallet/WalletCard.tsx [hunk 2]
```diff
@@ -30,50 +33,78 @@ export default function WalletCard({
...
+              ? '#004f4b'
+              : '#EAEAEA',
+        }}
+        className="main-card-container"
+      >
+        <div className="card-container">
+          <div className="icon-text">
+            <div className="icon">
+              <img src={smallLogo} alt="jhgasjg12i" />
+              {wallet?.credentialSubject[0].status !== 'Active' && (
```
Comment (nidhigarg-bmw) [inline src/components/pages/CompanyWallet/WalletCard.tsx:60]: same here as well

File: src/components/shared/MobileMenu/MenuItem.tsx [hunk 3]
```diff
@@ -91,8 +91,11 @@ export const MenuItem = ({
           typography: 'label3',
           textDecoration: 'none',
           whiteSpace: 'nowrap',
+          fontSize: '14px',
+          fontFamily:
```
Comment (nidhigarg-bmw) [inline src/components/shared/MobileMenu/MenuItem.tsx:95]: is it necessary to give particular font family ?

Comment (manojava-gk) [reply src/components/shared/MobileMenu/MenuItem.tsx:95]: yes

**Review by nidhigarg-bmw — APPROVED** [review]
lftm

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Comment by manojava-gk** [comment]
Conflicts resolved

**Review by oyo — APPROVED** [review]

**Pull request merged**